<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoardVlmNew</class>
 <widget class="QDialog" name="BoardVlmNew">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VLM Command Board</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>271</width>
     <height>341</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::East</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Compas</string>
    </attribute>
    <widget class="QLCDNumber" name="lcd_BS">
     <property name="geometry">
      <rect>
       <x>32</x>
       <y>10</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="value" stdset="0">
      <double>12.320000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="lab_BS">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>20</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>BS</string>
     </property>
    </widget>
    <widget class="tool_windAngle" name="windAngle" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>200</width>
       <height>200</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcd_TWS">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>300</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="value" stdset="0">
      <double>22.449999999999999</double>
     </property>
    </widget>
    <widget class="QLabel" name="lab_TWS">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>300</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TWS</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcd_TWD">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>300</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="digitCount">
      <number>6</number>
     </property>
     <property name="value" stdset="0">
      <double>117.239999999999995</double>
     </property>
    </widget>
    <widget class="QLabel" name="lab_TWD">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>300</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TWD</string>
     </property>
    </widget>
    <widget class="QLabel" name="lab_backTab1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>250</width>
       <height>341</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>lab_backTab1</zorder>
    <zorder>lcd_BS</zorder>
    <zorder>lab_BS</zorder>
    <zorder>windAngle</zorder>
    <zorder>lcd_TWS</zorder>
    <zorder>lab_TWS</zorder>
    <zorder>lcd_TWD</zorder>
    <zorder>lab_TWD</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Details</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Polaire</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>338</y>
     <width>252</width>
     <height>171</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <widget class="QDoubleSpinBox" name="spin_HDG">
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>110</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="suffix">
     <string>Â°</string>
    </property>
    <property name="maximum">
     <double>359.990000000000009</double>
    </property>
    <property name="value">
     <double>257.149999999999977</double>
    </property>
   </widget>
   <widget class="QLabel" name="lab_TWA">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>60</y>
      <width>29</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TWA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QDial" name="dial">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>50</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="sliderPosition">
     <number>1</number>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_VBVMG">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>39</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="lab_VMG">
    <property name="geometry">
     <rect>
      <x>219</x>
      <y>60</y>
      <width>29</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>VMG</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_ORTHO">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>102</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="lab_HDG">
    <property name="geometry">
     <rect>
      <x>121</x>
      <y>110</y>
      <width>29</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>HDG</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_VMG">
    <property name="geometry">
     <rect>
      <x>203</x>
      <y>61</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="lab_VBVMG">
    <property name="geometry">
     <rect>
      <x>157</x>
      <y>25</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>VBVMG</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_TWA">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>59</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="suffix">
     <string>Â°</string>
    </property>
    <property name="minimum">
     <double>-179.990000000000009</double>
    </property>
    <property name="maximum">
     <double>180.000000000000000</double>
    </property>
    <property name="value">
     <double>-154.319999999999993</double>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_TWA">
    <property name="geometry">
     <rect>
      <x>137</x>
      <y>61</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="lab_ORTHO">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>41</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>ORTHO</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_HDG">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>102</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_wp">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>140</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>WP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_pilototo">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>140</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Pilototo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_angleFlip">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>61</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string>background-color: rgb(239, 243, 247);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clearPilototo">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string>background-color: rgb(239, 243, 247);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clearWP">
    <property name="geometry">
     <rect>
      <x>227</x>
      <y>140</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string>background-color: rgb(239, 243, 247);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sync">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>29</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>VLM Sync</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_RANK">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lab_back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>250</width>
     <height>509</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>lab_back</zorder>
  <zorder>frame</zorder>
  <zorder>tabWidget</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tool_windAngle</class>
   <extends>QWidget</extends>
   <header>BoardTools.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
